app/scripts/modules/core/cluster/cluster.service.js

```diff
@@ -132,11 +132,23 @@ module.exports = angular.module('spinnaker.core.cluster.service', [
       'enableservergroup': baseTaskMatcher,
       'enablegoogleservergroup': baseTaskMatcher,
       'disablegoogleservergroup': baseTaskMatcher,
+      'rollbackServerGroup': function(task, serverGroup) {
+        var account = task.getValueFor('credentials'),
+            region = task.getValueFor('regions') ? task.getValueFor('regions')[0] : null;
+
+        if (account && serverGroup.account === account && region && serverGroup.region === region) {
+          return serverGroup.name === task.getValueFor('serverGroupName') ||
+            serverGroup.name === task.getValueFor('targetop.asg.enableServerGroup.name');
```

Comment:
Do we need a match for the `disableServerGroup` side?

enableServerGroup -> disableServerGroup?